### DISCUSSION TITLE
Choice of LXC Container OS

### DISCUSSION THREAD

**Discussion by yodaphone** [tteck/Proxmox#350] [Ideas]
I wanted to create a Docket LXC with your script but want to choose the 22.04 LTS as my container OS. the current script does not give me that option. Is this something thats possible if i have the container template in my repo?

**Comment by tteck** [comment]
Yep, change line 327 & 328 in `docker-v3.sh` to 
```
export PCT_OSTYPE=ubuntu
export PCT_OSVERSION=22.04
```
Also, you must use a password to avoid "Customizing Container" issues
